crmlogin/Contact.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+@brief Methods used to edit contacts. Used in actualContactEdit.php
+*/
+	class Contact{
+		
+
+		private $userInfo = array('contactId' => " ",'firstname' => " ", 'lastname' => " ", 'email' => " ");
+
+/**
+@brief Instantiates the object by contactId
+@param $id is the contactId of the contact that is being editted
+*/
+		public static function GetById($id){
+			include 'connection.php';
+
+			$sql = "SELECT * FROM `contactlist` WHERE `contactId`='$id'";
+			$contact = new Contact();
+			$result = $conn->query($sql);
+			$info = $result->fetch_assoc();
+
+			$contact->userInfo['firstname'] = $info['firstname'];
```

Comment:
Since the userInfo is private variables, this wouldn't work.

Use your helper methods, setValue and getValue